extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -137,6 +141,21 @@ export class AmpIframe extends AMP.BaseElement {
      * @private {!LayoutRect}
      */
     this.iframeLayoutBox_ = null;
+
+    /**
+     * The iframe may be in three states
+     * - HTMLIFrameElement, meaning an iframe has been created
+     * - undefined, meaning we have not yet created the iframe
+     * - null, meaning we will not have an iframe -- stop trying.
```

Comment:
This seems a tad cryptic. What if we just set `this.iframeSrc = null` when we can't create iframe. It's consistent with the current implementation and generally makes sense.